test/versioned/openai/mock-server.js

```diff
@@ -153,5 +153,9 @@ function getShortenedPrompt(reqBody) {
   const prompt =
     reqBody.prompt || reqBody.input || reqBody.messages.map((m) => m.content).join('\n')
 
+  if (typeof prompt === 'object') {
```

Comment:
```suggestion
  if (Object.prototype.toString.call(prompt) === '[object Object]') {
```

It's unlikely to happen, but I think it's better to guard against the following at all times so that no surprises creep in:

```js
if (typeof [] === 'object') {
  console.log('an object')
} else {
  console.log('not an object')
}
// an object
```

Reply:
I'll go with `Array.isArray` since we only need to check if it's an array